test/python/algorithms/evolvers/classical_methods/test_scipy_imaginary_evolver.py

```diff
@@ -0,0 +1,183 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test Classical Real Evolver."""
+import unittest
+from test.python.algorithms import QiskitAlgorithmsTestCase
+from ddt import data, ddt, unpack
+import numpy as np
+from qiskit.algorithms.evolvers.evolution_problem import EvolutionProblem
+from qiskit.opflow import I, X, Z, Zero, Plus, Minus, PauliSumOp
+from qiskit.opflow import StateFn, OperatorBase
+from qiskit import QuantumCircuit
+from qiskit.algorithms.evolvers.classical_methods import ScipyImaginaryEvolver
+
+
+@ddt
+class TestScipyImaginaryEvolver(QiskitAlgorithmsTestCase):
+    """Test Scipy Imaginary Evolver."""
+
+    def create_hamiltonian_lattice(self, num_sites: int) -> PauliSumOp:
+        """Creates an ising hamiltonian on a lattice."""
+        j_const = 0.1
+        g_const = -1.0
+
+        zz_op = ["I" * i + "ZZ" + "I" * (num_sites - i - 2) for i in range(num_sites - 1)]
+        x_op = ["I" * i + "X" + "I" * (num_sites - i - 1) for i in range(num_sites)]
+        return PauliSumOp.from_list(
+            list(zip(zz_op, len(zz_op) * [j_const])) + list(zip(x_op, len(x_op) * [g_const]))
+        )
+
+    @data(
+        [
+            Zero,
+            100,
+            X,
+            Minus,
```

Comment:
FYI If you remove the comma here, black will put it in one line, which might be a little nicer to read 🙂 
```suggestion
            Minus
```